Tests/Tests/AssetDataTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Nethereum.Hex.HexConvertors.Extensions;
+using ZeroX.Assets;
+using ZeroX.Utilities;
+
+namespace Tests.Tests
+{
+    [TestClass]
+    public class AssetDataTests
+    {
+
+        private static readonly EthereumAddress _testAddress = (EthereumAddress)Constants.TestEthereumAddress;
+        private static readonly byte[] _testERC20AssetData = ("0xf47261b0000000000000000000000000646934" +
+                "251e9045f41ddf7145dea5cd63cd45d8da").HexToByteArray();
+
+        private static readonly byte[] _testERC721AssetData = ("0x0257179200000000000000000000000064693425" +
```

Comment:
How much example coverage do you have? If it's only covering like 10% of the input data, then you can make an array of examples that you feed into every test (like input arguments).

Reply:
I will try to add more data in future PRs